websites/R/Raven Code/dist/metadata.json

```diff
@@ -0,0 +1,19 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.7",
+	"author": {
+		"name": "can",
+		"id": "613700645173592086"
+	},
+	"service": "Raven Code",
+	"description": {
+		"en": "For Raven Code web site.",
+		"tr": "Raven Code web sitesi için."
```

Comment:
This is not descriptive as well as the English description, maybe you can add more details, and please change "web sitesi" to "internet sitesi", it's more Turkish.